timed/reports/views.py

```diff
@@ -265,6 +268,13 @@ def _create_workreport(self, from_date, to_date, today, project, reports, user):
             str(13 + len(reports) - 1)
         )
 
+        # calculate location of total not billable hours as insert rows moved it
+        table[
+            13 + len(reports) + len(tasks) + 1, 2
+        ].formula = 'of:=SUMIF(C13:C{0};"nein";B13:B{0})'.format(
+            str(13 + len(reports) - 1)
+        )
+
```

Comment:
refactor this into (local) function as it does exactly the same as the lines above, except for a `+1` and `C` instead of `B`.

Reply:
It adds a new formula, but I can still refactor this into a function.

Reply:
Ah true, nevermind then.